static/app/views/performance/landing/widgets/components/performanceWidget.tsx

```diff
@@ -109,6 +122,9 @@ function _DataDisplay<T extends WidgetDataConstraint>(
             key={index}
             noPadding={Visualization.noPadding}
             bottomPadding={Visualization.bottomPadding}
+            onClick={() =>
+              trackDataComponentClicks(props.chartSetting, props.organization)
+            }
```

Comment:
Just checking, I think this will hit on any kind of click around the chart area, even ones where no action happens. Is this intended or should we be only tracking substantial events like zoom?

Reply:
Yes it will, and yeah it's intended. We're just trying to get an idea if the users are doing anything inside a widget.